onyo/commands/tests/test_get.py

```diff
@@ -152,6 +152,7 @@ def test_get_all(
 
     if machine_readable:
         for line in output:
+            line = [field.strip("\"") for field in line]
```

Comment:
IIRC you can use in cases like this the single quotes for strings, so as to not escape the double-quotes: `field.strip('"')`.
Obviously it is a matter of taste which one any given person finds more readable :)